3rdPartyNotices.txt

```diff
@@ -0,0 +1,26 @@
+Portions of this repo (The file SequenceOfT.cs) come from the https://github.com/AArnott/Nerdbank.Streams/ project,
+Exact commit : https://github.com/AArnott/Nerdbank.Streams/blob/23063b83512b3478848e514075e01cafed279955/src/Nerdbank.Streams/Sequence%601.cs
```

Comment:
This is good. I think you should include another license notice for MessagePack as well.

Reply:
I changed a little bit how the `3rdPartNotice` file is "scaffolded", is there a standard to respect.
Is that ok like this ?

Side question: Do you know why github seems to detect the `LICENSE` from `NerdBank.Streams` but not from `MessagePack` ?
https://github.com/AArnott/Nerdbank.Streams/blob/23063b83512b3478848e514075e01cafed279955/LICENSE
https://github.com/neuecc/MessagePack-CSharp/blob/2e421efb7bc53956fc66356565f9402d3271a588/LICENSE
![image](https://user-images.githubusercontent.com/2266487/73948949-f86a6400-48f9-11ea-9f07-cd93a25d4df5.png)

Reply:
(can you resolve this thread if it's ok for you ?)

Reply:
>  Do you know why github seems to detect the LICENSE from NerdBank.Streams but not from MessagePack ?

Yes: the messagepack one isn't the standard license text. It has two licenses in the same file.

Reply:
I don't see a "Resolve Conversation" button any more on this thread. But I would hit it if I could.

Reply:
(resolving from here then ^^ thx)